PlanningPoker.Entities.Tests/SessionTests.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using Xunit;
+
+namespace PlanningPoker.Entities.Tests
+{
+    public class SessionTests
+    {
+        [Fact]
+        public void Items_is_HashSet_of_Item()
+        {
+            var session = new Session();
+
+            var items = session.Items as HashSet<Item>;
+
+            Assert.NotNull(items);
+        }
+
+        [Fact]
+        public void Users_is_HashSet_of_User()
+        {
+            var session = new Session();
+
+            var users = session.Users as HashSet<User>;
+
+            Assert.NotNull(users);
```

Comment:
This isn't really testing that the type is a HashSet, is it? Can we test TypeOf?